src/app/modules/submission/encrypt-submission/encrypt-submission.controller.ts

```diff
@@ -259,8 +260,8 @@ const submitEncryptModeForm: ControllerHandler<
       break
     }
     case FormAuthType.SGID: {
```

Comment:
Would be helpful to also add one more case where

`FormAuthType.SGID_MyInfo` and return
`Storage mode form is not allowed to have SGID MyInfo authorisation`

Reply:
Ah completely overlooked this part. Thanks!